hc/api/tests/test_ensuretriggers.py

```diff
@@ -14,16 +14,19 @@ def test_ensure_triggers(self):
 
         check = Check.objects.create()
         assert check.alert_after is None
+        self.assertIsNone(check.alert_after, msg=None)
 
         check.last_ping = timezone.now()
         check.save()
         check.refresh_from_db()
         assert check.alert_after is not None
-        ### The above assert fails. Make it pass
+        self.assertIsNotNone(check.alert_after, msg=None)
 
         alert_after = check.alert_after
 
         check.last_ping += timedelta(days=1)
         check.save()
         check.refresh_from_db()
-        ### Assert that alert_after is lesser than the check's alert_after 
+        alert_after_refresh = check.alert_after
+        ### Checks that alert_after is lesser than the check's alert_after
```

Comment:
remove this prompt. otherwise good

Reply:
Fixed. Thank you for that, had missed it.